### PR TITLE
New challenges

### PR DESCRIPTION
**Pull request by mka2910**
New challenges wrote. The to the moon challenge shouldn't be too difficult now.

### CODE REVIEW

File: do-challenges.html [hunk 1]
```diff
@@ -76,7 +83,20 @@
...
 
-    <p><b>Newest challenge:</b> Insult Generator, in <a href="#insult-generator-scratch">Scratch</a> or in <a href="#insult-generator-coding">Code</a>.</p>
+    <p>
+      <b>Newest challenges:</b>
+      <br>
+      <p>
+        Scratch:
+        <a href="#to-the-moon">To the moon</a>
+      </p>
+      <p>
```
Comment (greghuc) [inline do-challenges.html]: Need link to coding challenge

File: do-challenges.html [hunk 2]
```diff
@@ -417,6 +437,48 @@ <h3 name="wind-whistle">Challenge: Whistle for the Wind</h3>
...
 
+                <a name="to-the-moon"></a>
+                <h3>Challenge: To the Moon</h3>
+                <p>
+                  Write a game about flying to the moon. There should be three stages in your
+                  game. One set on Earth; one in space and one on the moon.
+                </p>
+                <p>
+                  To start the rocket the player should press the space bar five times. Then the
+                  rocket should fly into the air. When the rocket reaches the edge of the screen,
```
Comment (greghuc) [inline do-challenges.html]: "edge" -> "top"?

File: do-challenges.html [hunk 3]
```diff
@@ -417,6 +437,48 @@ <h3 name="wind-whistle">Challenge: Whistle for the Wind</h3>
...
+                  the background should be changed to the next background (e.g. from Earth to space). Along with placing the
+                  rocket at the bottom of the screen again. The player should be
+                  able to move the rocket left and right using the arrow keys. There should be
+                  Aliens and other space-ships flying around, if the player touches one of these
+                  they lose the game. When the player reaches the last stage (the moon), a target
```
Comment (greghuc) [inline do-challenges.html]: When the rocket enters the final moon screen, is the moon (target) at the top or the bottom of the screen? Was wondering this, as the previous screens have the rocket going from the bottom to the top (lift off and travel though space)

File: do-challenges.html [hunk 4]
```diff
@@ -417,6 +437,48 @@ <h3 name="wind-whistle">Challenge: Whistle for the Wind</h3>
...
+                      For a spaceship sprite look in the sprite library (they'll be found in flying)
+                    </p>
+                  </li>
+
+                  <li>
+                    <p>
+                      To count the number of times the player has pressed space make a variable
+                      (found under data).
+                      Then every time space is pressed add one to this variable. Use the wait until
+                      block to make the sure the rocket doesn't fly off until the variable equals ten.
```
Comment (greghuc) [inline do-challenges.html]: Be consistent on the variable count. It's 5 above, and 10 here.

File: do-challenges.html [hunk 5]
```diff
@@ -603,6 +665,36 @@ <h3 name="favourite-films">Challenge: My Favourite Films</h3>
                 I'm looking forward to what you come up with :-) Thanks, Bob"
                 </p>
 
+                <h3>Challenge</h3>
```
Comment (greghuc) [inline do-challenges.html]: Need a challenge title

**Comment by mka2910** [comment]
I think I have made all the necessary changes

**Comment by greghuc** [comment]
Cool. Done. Merged. Thanks :-)

**Pull request merged**